### DISCUSSION TITLE
Alexa Skill Devices

### DISCUSSION THREAD

**Discussion by xPudda** [alandtse/alexa_media_player#1531] [Q&A]
Hi guys, 

since I no longer know who to ask for advices on how to solve my question, I am asking it here too, maybe someone can give me an answer. 
I have a skill in my amazon account that allows me to control home automation (shutters, thermostats) from the Alexa app.
In the app i saw all these devices and i can normally control them.
Now... I wish I could also see these devices in Home Assistant (which I repeat, they are only connected and connectable via skill, as they pass through the brand's cloud) to make some automations. 

So, Alexa Skill Devices -> Home Assistant. 

Alexa should be my proxy.

How the hell do I do that? Actually Alexa Media Player is showing me up only the echo devices connected to my account, as it should be.

Thanks in advance.. You could save me and my soul.. :)

**Comment by Kurisudes** [comment]
Did you try costum command?
[https://github.com/custom-components/alexa_media_player/wiki#run-custom-command](url)
